central/cloudsources/datastore/telemetry.go

```diff
@@ -43,11 +47,13 @@ func Gather(ds DataStore) phonehome.GatherFunc {
 			return len(cloudSources), nil
 		})
 
-		totalCount := map[string]int{}
+		totalCount := map[string]int{
+			trimPrefix(storage.CloudSource_TYPE_OCM.String()):           0,
```

Comment:
Do we need to handle `TYPE_UNSPECIFIED`?

Reply:
for future proof we can iterate over `enumregistry.GetComplementByExactMatches` but I think it's not needed right now

Reply:
looped over the types